spec/features/smart_answer/find_coronavirus_support_flow_spec.rb

```diff
@@ -0,0 +1,249 @@
+require "rails_helper"
+
+RSpec.feature "FindCoronavirusSupportFlow", type: :feature do
+  def start_flow_to_need_help_with
+    visit "/find-coronavirus-support"
+    expect(page).to have_selector("h1", text: headings[:flow_title])
+
+    click_govuk_start_button
+    expect(page).to have_selector("h1", text: headings[:need_help_with])
+  end
+
+  def continue_to_results
+    click_button "Continue"
```

Comment:
This relies on the button being `Continue`.  However, there is the option to change the label on that button now.  Is there a way to mitigate having to update all occurrences of `Continue` in this file when someone changes the button text?  Could we use an ID or `locale` files?

Reply:
"Continue" isn't the default button text. The default is "Next step".

OK - so say we use a translation file here. We have two next step labels, which need identifying with a key. I'd suggest `:continue` and `:next_step`. The code then becomes something like:

```ruby
click_button i18n.t('flow.next_button.continue')
```
If we change the button back to default we'd need to change each occurrence to:

```ruby
click_button i18n.t('flow.next_button.next_step')
```

We could wrap it all in a method. So we have something like:

```ruby
click_next_button(:continue)
```
But then we've obscured what is actually going on by one step - a step that I'd argue is unnecessary.

I am very conscious of the way RSpec feature tests have been implemented in other GDS projects; where everything is wrapped up in methods in what I call pseudo-cucumber. It makes it very hard to work out what the tests are actually doing. This is especially annoying when the standard RSpec and Capybara syntax is so descriptive without this unnecessary obscuration.

So I would argue that considering all the options, the clearest and simplest solution is just to use:
```ruby
click_button "Continue"
```
Everyone familiar with Capybara should understand what it's doing, and it is just as simple to change "Continue" to "Next Step" as it is `:continue` to `:next_step`.

Sure - where the text is long, use translation files. But for very basic actions like this, I'd argue that the simplest solution is the best.

Reply:
I was hoping we could do something like this...

```
# config/locales/en.yml
next_button: "Next step"
...
flows:
  find_support: 
    next_button: "Continue"
```

Then ...
```
# lib/smart_answer_flows/find-coronavirus-support.rb
module SmartAnswer
  class FindCoronavirusSupportFlow < Flow
    def define
      # ...
      button_text i18n.t('flows.find_support.next_button')
```